Lib/asyncio/test_utils.py

```diff
@@ -361,6 +361,13 @@ def assert_writer(self, fd, callback, *args):
             handle._args, args)
 
     def _ensure_fd_no_transport(self, fd):
+        if not isinstance(fd, int):
+            try:
+                fd = int(fd.fileno())
+            except (AttributeError, TypeError, ValueError):
+                # This code matches `selectors._fileobj_to_fd` function.
```

Comment:
Avoid backticks.

Reply:
Why?

Reply:
@1st1 I hate any complex markup in comments, `selectors._fileobj_to_fd` without backticks is still pretty readable and understandable.
This is just my opinion.
But if you want to keep them -- I can live with it. Very minor thing about a taste.